src/cargo/sources/registry/http_remote.rs

```diff
@@ -591,9 +591,14 @@ impl<'cfg> RegistryData for HttpRegistry<'cfg> {
         }
         if self.auth_required {
             self.check_registry_auth_unstable()?;
-            let authorization =
-                auth::auth_token(self.config, &self.source_id, self.login_url.as_ref())?;
-            headers.append(&format!("Authorization: {}", authorization))?;
+            let credential = auth::auth_token(self.config, &self.source_id)?;
+            let authorization = credential.as_header(
+                self.config,
+                &self.source_id,
+                self.login_url.as_ref(),
+                None,
+            )?;
+            headers.append(&format!("authorization: {}", authorization))?;
```

Comment:
Keep the capitalized `Authorization`, otherwise the log-redaction doesn't work.

```suggestion
            headers.append(&format!("Authorization: {}", authorization))?;
```

Reply:
Oops.. will be fixed in the next push.